boards/clue_nrf52840/src/main.rs

```diff
@@ -355,6 +361,28 @@ pub unsafe fn reset_handler() {
 
     kernel::hil::sensors::ProximityDriver::set_client(apds9960, proximity);
 
+    let sht3x = components::sht3x::SHT3xComponent::new(sensors_i2c_bus, mux_alarm).finalize(
+        components::sht3x_component_helper!(
+            nrf52::rtc::Rtc<'static>,
+            capsules::sht3x::BASE_ADDR << 1
+        ),
+    );
+    sht3x.reset();
+
+    let temperature = static_init!(
+        capsules::temperature::TemperatureSensor<'static>,
+        capsules::temperature::TemperatureSensor::new(sht3x, board_kernel.create_grant(&grant_cap))
+    );
+
+    kernel::hil::sensors::TemperatureDriver::set_client(sht3x, temperature);
```

Comment:
We do have a temperature sensor capsule if you want to use that.

Reply:
we use it

Reply:
I am guessing Brad meant to say "component" not "capsule"

Reply:
I fixed this and I added humidity component also.